### DISCUSSION TITLE
Runtime export for WebGL

### DISCUSSION THREAD

**Discussion by Tripl3Design** [atteneder/glTFast#361] [General]
Really cool to see that it's now possible to export models at runtime!
Is it also possible to export a model to a webserver in a WebGL environment?
I did something in the past with MultipartFormSection and UnityWebRequest.

**Comment by atteneder** [comment]
Hi @Tripl3Design and thanks!

I have'nt tried, but it should be possible, yes. In the last release I added the API to export to a (Memory) `Stream`. You can use networking APIs to upload said data to a server.

Let me know if you have success with it.
